keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -172,11 +203,34 @@ public class KeeperPlugin : Plugin<Project> {
                   }
                 }
               }
+
+          // Now clear the outputs from androidTest's L8 task to end with
+          tasks
+            .named<L8DexDesugarLibTask>(interpolateL8TaskName(testVariant.name))
+            .configure {
+              doLast {
+                clearDir(desugarLibDex.asFile.get())
+              }
+            }
         }
       }
     }
   }
 
+  private fun clearDir(path: File) {
+    if (!path.isDirectory) {
```

Comment:
The kdoc for `File::deleteRecursively` makes it sound like it handles both files and directories -- could this be simplified?

```
if (path.exists()) {
  path.deleteRecursively()
}
if (!path.mkdirs()) {
  throw IOException(String.format("Could not create empty folder %s", path))
}
```

Reply:
I supposed we could do that but I assumed there was a reason it didn't try to delete the directory if it was already present (maybe gradle snapshotting?). Would rather go with this for now and check as a followup